server/model/display-type.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+export type DisplayType = 
+  | "lead"
+  | "regular";
```

Comment:
`weight` is more about what relation this piece of content (not only images but galleries, video, text etc) has in relation to it's context.